### PR TITLE
Add repository type and endpoints. Add pull_requests to Story type.

### PR DESCRIPTION
**Pull request by feeihoa**

### CODE REVIEW

**Review by hubjacek — CHANGES_REQUESTED** [review]
20

**Review by hubjacek — COMMENTED** [review]
src/index.ts

**Review by hubjacek — COMMENTED** [review]
10

**Review by hubjacek — APPROVED** [review]

File: src/types.ts [hunk 1]
```diff
@@ -376,13 +376,14 @@ export type Story = {
...
   project_id: number;
+  pull_requests: Array<PullRequest>;
   requested_by_id: ID;
   started: boolean;
   started_at: string | null;
   started_at_override: string | null;
   story_type: StoryType;
   tasks: Array<Task>;
-  tasks_id: Array<ID>;
+  task_ids: Array<ID>;
```
Comment (charpeni) [inline src/types.ts:386]: Nice catch!

**Review by charpeni — APPROVED** [review]
Thank you!

**Pull request merged**